src/Config.cpp

```diff
@@ -77,6 +79,30 @@ void Config::readFromFile(const string &file) {
     }
 }
 
+void Config::readFromMap(map<string, any> &mp) {
+    bool given = false;
+    string CTL = "column_table_limit";
+    string CL = "column_limit";
+    for (auto kv : mp) {
+        auto key = kv.first;
+        if (node_[key]) {
+            if (strcmp(kv.second.type().name(), "i") == 0) {
```

Comment:
`strcmp(kv.second.type().name(), "int")`

Reply:
it returns "i" not  "int"

Reply:
OK